corehq/apps/app_manager/app_schemas/form_metadata.py

```diff
@@ -238,9 +238,11 @@ def __init__(self, app1, app2):
         self.first = get_app_summary_formdata(app1.domain, app1)[0]
         self.second = get_app_summary_formdata(app2.domain, app2)[0]
 
-        self._first_by_id = {}
+        self._first_modules_by_id = {}
+        self._first_forms_by_id = {}
```

Comment:
this change ended up not really mattering, but I think this is clearer, since we always know whether an ID corresponds to a form or a module.

Reply:
agreed that this is clearer now